.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,30 @@
+name: Pre-commit checks
```

Comment:
Should this explicitly mention "Husky" so we don't confuse it with the Python pre-commit tool?